### PR TITLE
Change coordinate used in newly added `ScrollingHitObjectContainer` methods

### PR DESCRIPTION
**Pull request by ekrctb**
The PR #13449 is already merged, but after more thinking and coding, using the coordinate that satisfies `0 = current time` (e.g. `Bottom` anchor for `Down` scrolling) is more convenient than using the `TopLeft` anchor.
My previous PR was motivated by an incorrect code that is breaking when the scroll is inverted.

I hope it is okay to change the public methods after just added, as they are not used by anyone yet.

### CODE REVIEW

File: osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs [hunk 1]
```diff
@@ -81,8 +81,10 @@ public double TimeAtPosition(float localPosition, double currentTime)
         /// </remarks>
         public double TimeAtScreenSpacePosition(Vector2 screenSpacePosition)
         {
-            Vector2 localPosition = ToLocalSpace(screenSpacePosition);
-            return TimeAtPosition(scrollingAxis == Direction.Horizontal ? localPosition.X : localPosition.Y, Time.Current);
+            Vector2 pos = ToLocalSpace(screenSpacePosition);
+            float localPosition = scrollingAxis == Direction.Horizontal ? pos.X : pos.Y;
+            localPosition -= axisInverted ? scrollLength : 0;
```
Comment (bdach) [inline osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs:86]: I guess this transformation seems fine math-wise, but I'm having trouble on how to interpret this value. It looks like it's the relative position of the object across the scrolling axis from the near edge of the scroll... but with the sign in screen coordinates, not the scroll coordinates...?

Might be a good idea to explain that in the xmldoc of `TimeAtPosition` or something...

**Comment by bdach** [comment]
As for whether it's fine to change behaviour of these public methods, I do think it's definitely fair game to change them without it being a breaking change if they haven't actually been included in a release yet.

File: osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs (hunk 1 above)
Comment (ekrctb) [reply osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs:86]: I found that it is hard to explain the thing without mentioning the implied coordinate system (anchoring bottom for down scroll etc.).
But I don't more changes to be included in PR.

**Review by smoogipoo — APPROVED** [review]
Sounds fair to me

**Pull request merged**